src/ray/common/ray_config_def.h

```diff
@@ -434,6 +434,9 @@ RAY_CONFIG(int32_t, gcs_client_check_connection_status_interval_milliseconds, 10
 
 /// Feature flag to use the ray syncer for resource synchronization
 RAY_CONFIG(bool, use_ray_syncer, false)
+/// Due to the protocol draw back, raylet needs to refresh the message if
+/// no message is received for a while.
+RAY_CONFIG(int64_t, message_refresh_interval_ms, 3000)
```

Comment:
maybe name it `ray_syncer_message_refresh_interval_ms` for now until ray syner is rolled out.

Reply:
Also add a link to the protocol draw back for future reference.

Reply:
I think we need this until the protocol is updated.

We'll roll out ray syncer with this config.

I'll add comment and rename it.